.github/workflows/test-publish.yml

```diff
@@ -1,7 +1,9 @@
-name: Node Build / Test / Lint / Publish
+name: Test and Publish
```

Comment:
The diff for this PR is wacky.  Really this is a new file and I renamed/modified what was `built-test-lint-publish.yml` to `lint-test.yml` but somehow GitHub mixed the two up...